analytic_partner/__manifest__.py

```diff
@@ -6,7 +6,7 @@
 {
     "name": "Link analytic items and partner",
     "summary": "Search and group analytic entries by partner",
-    "version": "14.0.1.0.0",
+    "version": "16.0.0.1.0",
```

Comment:
`16.0.1.0.0`

Reply:
hello, thanks you for your review.

I'll squash my future commit no problem.

regards,